src/Jobs/Recruit.Vacancies.Jobs/NonLevyAccountBlocker/NonLevyAccountBlockerJob.cs

```diff
@@ -39,6 +43,30 @@ public async Task RefreshBlockedEmployerAccountsAsync([TimerTrigger(Schedules.Ho
 
             _logger.LogInformation("Starting rebuilding blocked employers reference data.");
 
+            List<string> blockedEmployers;
```

Comment:
👍  OK I'll use the disabledJobs instead.  

I don't understand the reason why we are scoping feature toggles.  Toggles are normally cross cutting and can be used anywhere. The equivalent of commenting out code.

Reply:
I see features as something that affects the user and how they interact with the consuming system. We are only using boolean feature toggles at the moment but anything else controlling system behaviour would be a boolean  toggle that is not related to a feature.

EnableLevy sounds like something that seems we only care about relating to the employer site. We have a mixture of concerns in that we are using orchestrators and then have a core classes wrapped in client which has the application layer containing the commands (things that are operations from the separate systems) so it is harder to see the line. I am struggling to think of when we would have a feature toggle that needs inspection from within the client lib. I'm probably being pedantic, happy to consider/understand where this might be needed. I see you have moved the Feature toggle inspection service to the client, I don't mind if you don't switch that back if you think it lives better there.

Reply:
Thanks. I'm not sure I'm fully on the same page. But now that the job will rely on the blockedJobs list I can move the toggle functionality back to Shared Web.  We can have another think when there is a need to change behaviour in the jobs :)